recipes/graylog-forwarder/files/pre-install

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+
+set -e
+
+user="graylog-forwarder"
+group="graylog-forwarder"
+datadir="/var/lib/graylog-forwarder"
+
+case "$1" in
+	# DEB based systems
+	install|upgrade)
+		if ! getent group "$group" > /dev/null 2>&1 ; then
+			addgroup --system "$group" --quiet
+		fi
+
+		if ! id "$user" > /dev/null 2>&1 ; then
+			adduser --system --home "$datadir" --no-create-home \
+				--ingroup "$group" --disabled-password \
+				--gecos "Graylog Forwarder" \
+				--shell /bin/false --quiet "$user"
+		fi
+		;;
+	abort-deconfigure|abort-upgrade|abort-remove)
+		# Ignore
+		;;
+	# RPM based systems
+	1|2)
+		# Create graylog group
+		if ! getent group "$user" >/dev/null; then
+			groupadd -r "$user"
```

Comment:
Doesn't make a difference because it's the same, but I think this should use the group variable:
```suggestion
		if ! getent group "$group" >/dev/null; then
			groupadd -r "$group"
```

Reply:
Fixed.